.travis.yml

```diff
@@ -39,6 +39,10 @@ script:
       mvn clean install -B -U
     fi
 
+# enable codecov-io
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
this seems.. unsafe.  Is there no direct integration we can provide? Does it have to be through Travis?